project/AppDependencies.scala

```diff
@@ -21,16 +21,16 @@ import sbt.*
 object AppDependencies {
 
   val bootstrapVersion         = "8.4.0"
-  val playVersion               = "play-28"
-  val playPartialsVersion      = s"8.4.0-$playVersion"
-  val hmrcMongoVersion         = "1.3.0"
+  val playVersion               = "play-30"
+  val playPartialsVersion      = "9.1.0"
```

Comment:
Just an observation: I don't see a point in having `val playPartialsVersion` as that's used only once

Reply:
I noticed this as well, decided to leave it in as it was added by a talented frontend dev and I didn't want to go over his work as it's functional anyway - but yes I agree that it seems redundant